vmngclient/api/task_status_api.py

```diff
@@ -102,10 +32,20 @@ def __init__(self, session: vManageSession, task_id: str):
         self.url = f"/dataservice/device/action/status/{self.task_id}"
         self.task_data: List[SubTaskData]
 
+    def __validate_if_failure(self, wait_seconds: int = 5):
```

Comment:
I think `__check_validation_status` is better name.
Current name suggest that we are validating something when something failed.